app/routers/users-group/index.js

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
+
+import { addUsersToGroup } from './addUsersToGroup';
+import { validator } from '../../validation/usersGroupValidation';
+
+const usersGroupRouter = Router();
+
+usersGroupRouter.post('/users-group', validator, addUsersToGroup);
```

Comment:
In my opinion it's better to use `/users-group` route, because here we can assign an array of users to one specific group. Also function name 'add**Users**To**Group**' suggests that there may be several users, but group only one.